setup.py

```diff
@@ -40,6 +40,9 @@
     install_requires=[
         'chardet>=3.0.2',
     ],
+    tests_require=[
```

Comment:
Test require is now going out of style. Do you mind to remove this?

Reply:
Should you decide to merge the PR, I'll remove it.